apps/frontend/src/pages/home/components/nav/nav.tsx

```diff
@@ -0,0 +1,14 @@
+import { type JSX } from "react";
+
+import { Link } from "~/libs/components/components.js";
+
+const Nav = (): JSX.Element => {
```

Comment:
let's type Nav as React.FC and remove JSX.Element return type. Make similar changes in the whole your PR